webpages/styles/colors-light.css

```diff
@@ -1,5 +1,4 @@
 :root {
-  --brand-orange-min-font-weight: 600;
```

Comment:
This can now be readded.

Reply:
Okay, but why is `--brand-orange-min-font-weight` 600 in light and `normal` in dark? For me, it makes certain text bold only when I'm in light mode. That's why I removed this line initially.

Reply:
The variable is used for these links:
![Mark messages as read | Open messages page](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/51b3ba91-a6c8-42a5-aa99-8d4152f7a97e)

They become orange on hover, which makes them hard to read in light mode. The bold font makes it slightly better. It would be even better to use a different color (probably `--blue-text`), but that should be a separate PR.